cli/root.go

```diff
@@ -108,12 +108,13 @@ func AGPL() []*cobra.Command {
 }
 
 func Root(subcommands []*cobra.Command) *cobra.Command {
+	fmtLong := `Coder v%s — A tool for provisioning self-hosted development environments with Terraform.
```

Comment:
```suggestion
	fmtLong := `Coder %s — A tool for provisioning self-hosted development environments with Terraform.
```

Remove the duplicate `v`

Reply:
Removed in the latest commit [`84be5c7`](https://github.com/coder/coder/pull/4709/commits/84be5c752226ae6849bf1a59166801878a13a662)